### PR TITLE
fix #6034 chore(nimbus): remove locale/country filtering from targeting configs

### LINKED ISSUE #6034: Remove locale filtering from targeting configs
Now that we have the ability to explicitly set locale/country filtering, we no longer need locale/country filtering baked into the targeting configs.  Let's update the targeting configs to remove any locale/country specific filtering.

┆Issue is synchronized with this [Jira Task](https://mozilla-hub.atlassian.net/browse/EXP-1533)

### PR DESCRIPTION
**Pull request by jaredlockhart**
Becuase

* We can now explicitly target by locale/country independently of targeting configs
* Allowing locale/country filtering in targeting configs in addition to the specific fields can create invalid targeting expressions

This commit

* Removes all locale/country filtering from targeting configs

### CODE REVIEW

**Comment by jaredlockhart** [comment]
@pdehaan Do you wanna see if you can validate these updated targeting expressions locally in Firefox if you have the time?

File: app/tests/integration/nimbus/test_e2e_create_experiment.py [hunk 1]
```diff
@@ -40,7 +40,7 @@ def test_create_new_experiment(selenium, base_url):
     audience = metrics.save_and_continue()
     audience.channel = "Nightly"
     audience.min_version = 80
-    audience.targeting = "US_ONLY"
+    audience.targeting = "TARGETING_MAC_ONLY"
```
Comment (jrbenny35) [inline app/tests/integration/nimbus/test_e2e_create_experiment.py:43]: Need to make this fixtures but that's my job not yours :)

**Review by jrbenny35 — APPROVED** [review]

**Review by lmorchard — COMMENTED** [review]
LGTM :+1:

**Review by lmorchard — APPROVED** [review]
Er, "approve" is what I meant to hit!

**Pull request merged**